notion_client/client.py

```diff
@@ -44,7 +48,7 @@ class ClientOptions:
         notion_version: Notion version to use.
     """
 
-    auth: Optional[str] = None
+    auth: Optional[Union[str, tuple[str, str]]] = None
```

Comment:
The JS SDK doesn't support this kind of auth at this level AFAICT, so this should be reverted.